src/autoevals/base.py

```diff
@@ -45,7 +43,7 @@ async def _run_eval_async(self, output, expected=None, **kwargs) -> Evaluation:
         return self._run_eval_sync(output, expected, **kwargs)
 
     @abstractmethod
-    async def _run_eval_sync(self, output, expected=None, **kwargs) -> Evaluation:
+    def _run_eval_sync(self, output, expected=None, **kwargs) -> Evaluation:
```

Comment:
@ankrgyl - Wanted to double flag this and ensure the removal of `async` from this eval method makes sense.

Reply:
Yes please this is a typo 🤦